codegen.ts

```diff
@@ -0,0 +1,34 @@
+import type { CodegenConfig } from '@graphql-codegen/cli';
```

Comment:
_Random placement for comment_


The deploy preview Vote page seems broken.
<img width="1033" alt="image" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/91209fee-a748-4b43-a417-37d607244317">
Console:
<img width="737" alt="image" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/faabe4d4-5525-4af1-9364-2c02d448591b">

Reply:
Thanks for taking a look! Yes, there looks to be some weirdness going on. I am going to take a deeper look now.

Reply:
I think this may fix it https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/177/commits/a91114957dccf740a51d5a22e15cf355bfee887d